### PR TITLE
MOD-2616 (UT) - Added unit test cases for image overlay and circle proxy

### PR DESCRIPTION
**Pull request by anil-shukla-axway**
https://jira.appcelerator.org/browse/MOD-2616

### CODE REVIEW

File: test/unit/specs/methods.spec.js [hunk 1]
```diff
@@ -69,10 +70,131 @@ describe('ti.map', function () {
...
+						type: Map.POLYLINE_PATTERN_DASHED,
+						gapLength: 25,
+						dashLength: 50
+					});
+
+					expect(Map.setCenter(pattern)).toEqual(jasmine.any(Function));
+				});
+			});
+			// TiMapSnapShotterProxy
+			describe('setRegion', () => {
```
Comment (vikas-goyal-axway) [inline test/unit/specs/methods.spec.js:157]: @anil-shukla-axway  why did you add Snapshotterproxy test case in this pr. Snapshotterproxy is covered in different ticket so it should have its own pr.

File: test/unit/specs/methods.spec.js [hunk 2]
```diff
@@ -69,10 +69,27 @@ describe('ti.map', function () {
...
-		// TDOO: Test actually adding an annotation to a view
+			describe('setBoundsCoordinate', () => {
+				it('to set bound coordinates',  () => {
+					var fileSystem = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, 'images', 'flower.jpg.keep');
+					var newBlob = fileSystem.read();
+
+					expect(Map.setBoundsCoordinate(newBlob)).toEqual(jasmine.any(Function));
+				});
+			});
+			// TDOO: Test actually adding an annotation to a view
```
Comment (fasihul-axway) [inline test/unit/specs/methods.spec.js:91]: @anil-shukla-axway, did you verified this TODO?

Comment (anil-shukla-axway) [reply test/unit/specs/methods.spec.js:91]: No I did not verified this.

Comment (fasihul-axway) [reply test/unit/specs/methods.spec.js:91]: Please verify and and update PR if required.

**Comment by sgtcoolguy** [comment]
I think more tests is better than none, even if there's TODOs left in here. I'm going to merge this.
